compiler/include/driver.h

```diff
@@ -168,6 +168,8 @@ enum { LS_DEFAULT=0, LS_STATIC, LS_DYNAMIC };
 
 extern int  fLinkStyle;
 
+extern bool fGeneratePIC;
```

Comment:
I think that it'd be better to add CHPL_LIB_PIC to this header, not add fGeneratePIC at all, and have the code in clangUtil that was using fGeneratePIC do the `(strcmp(CHPL_LIB_PIC, "pic") == 0)`. This way, the variable has a well-known name (since it's used elsewhere in the compiler). This will help make it easier for us to understand what is impacted by setting that variable.

Also, should CHPL_LIB_PIC have a command-line flag, the same way that we have `--target-platform` for `CHPL_TARGET_PLATFORM`?